res/css/structures/_LeftPanel.scss

```diff
@@ -14,7 +14,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-$roomListCollapsedWidth: 68px;
+:root {
+    --room-list-collapsed-width: 68px;
+}
```

Comment:
@gsouquet postcss does smart compliation, it uses the variable directly with it's value and then it overrides with the value references, e.g.
```
:root {
  --my-var: 1px
}

.myClass {
  width: var(--my-var);
}
```
will get compiled to 
```
:root {
  --my-var: 1px
}

.myClass {
  width: 1px;
  width: var(--my-var);
}
```

This change is a necessary part of this pull request, as this is directly being used in javascript, so that we don't double the logic.